### PR TITLE
add layout and fileUpload page

### PR DESCRIPTION
**Pull request by ghost**
Please checkout my branch and do some tests.

Thx! ;-)

### CODE REVIEW

File: src/app/sl-public/components/sl-import/sl-import.component.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          <th>Progress</th>
+          <th>Status</th>
+          <th>Source</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let item of uploader.queue">
+          <td valign="middle">
+            <strong *ngIf="checkFileExt(item.file.type);" class="error-import-info">{{item?.file?.name}}</strong>
```
Comment (ghost) [inline src/app/sl-public/components/sl-import/sl-import.component.html:39]: Maybe here is some a more beautiful solution possible... but I didn't find one...

Comment (samsamann) [reply src/app/sl-public/components/sl-import/sl-import.component.html:39]: validate the files in the controller or use directives. but I think it's ok at the moment.

**Review by samsamann — COMMENTED** [review]
clean up unused dependencies

**Comment by ghost** [comment]
@samsamann, please have again a look on it. And when everything is OK merge it, when NOK contact me ;-)

**Pull request merged**